docker/Dockerfile.dev

```diff
@@ -19,6 +19,10 @@ ARG BRANCH_NAME=master
 ARG MACHINE_TYPE=cpu
 ARG CUDA_VERSION
 
+ARG BUILD_WITH_IPEX
```

Comment:
should we also include the launcher script stuff as well in this PR?

Reply:
OK, do you mean, for example, jemalloc, iomp, etc ?